### DISCUSSION TITLE
I'm not a wildlife recorder

### DISCUSSION THREAD

**Discussion by fruit-github** [FieldStudiesCouncil/QGIS-Biological-Recording-Tools#66] [General]
I'm not a wildlife recorder, I don't have sufficient knowledge or skill - though I do have many FSC foldouts. 

I have found this such a useful plugin and hadn't given any thought to praising it here until QGIS 3.30 broke it. Thankfully it's now fixed and working again.

I volunteer with a few local heritage, archaeology, forestry and history groups and create many maps for use in their projects. The ability to place  and pick points and mark out grids using OSGB coords makes life so much easier

Thanks to all involved

**Comment by mhanmore** [comment]
Great post!  I've just discovered the plugin and also finding it really useful. In my case working with Environmental Permits and Water Abstraction Licenses which often use 10m grid locations, so the clipboard-to-polygon-square button is hugely useful. Thanks devs!
